spec/factories/recipe.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  sequence :name do |n|
+    "chicken#{n}"
+  end
+
+  factory :recipe do
+    name
+    association :user
```

Comment:
I had to look this up. The `association` here is optional, it works implicitly since the factory and association names are the same https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#implicit-definition